Umbraco-Cloud/Upgrades/Minor-Upgrades/index.md

```diff
@@ -5,45 +5,48 @@ versionTo: 10.0.0
 
 # Minor version upgrades
 
-When minor version upgrades are available, your Umbraco site will not be auto-upgraded to this version. You will need to press the Upgrade button in the Umbraco Cloud portal to perform the upgrade. This will upgrade your Development environment so you can test how everything works on a Cloud environments before pushing the upgrade to your Live site.
+When minor version upgrades are available, your Umbraco site will not be auto-upgraded to this version. You will need to press the Upgrade button in the Umbraco Cloud portal to perform the upgrade. This will upgrade your Development environment so you can test how everything works on a Cloud environment before pushing the upgrade to your Live site.
 
-This workflow applies to all producs on Umbraco Cloud: Umbraco CMS, Umbraco Forms and Umbraco Deploy.
+This workflow applies to all products on Umbraco Cloud: Umbraco CMS, Umbraco Forms, and Umbraco Deploy.
+
+For Starter plans, you will need to add a Development environment first before you can perform the semi-automatic upgrade. Find pricing details for Umbraco Cloud Starter plans on our
```

Comment:
Is this a duplicate line of the one below?

Reply:
it totally is! let me just remove it :D